src/ansys/fluent/core/utils/fluent_version.py

```diff
@@ -1,9 +1,27 @@
+from enum import Enum
+from functools import total_ordering
 import os
 from typing import Optional
 
 import ansys.fluent.core as pyfluent
 
 
+class AnsysVersionNotFound(RuntimeError):
+    """Provides the error when Ansys version is not found."""
+
+    def __init__(self):
+        super().__init__("Verify the value of the 'AWP_ROOT' environment variable.")
```

Comment:
Nitpick: we are only raising `AnsysVersionNotFound` in a single location. Would it make more sense to go with:
```suggestion
class AnsysVersionNotFound(RuntimeError):
    """Provides the error when Ansys version is not found."""
    pass
```
and when raising the exception:
`raise AnsysVersionNotFound("Verify the value of the 'AWP_ROOT' environment variable.")`
Less lines and cleaner in my opinion.

I think it only really makes sense to consolidate the error message as you suggest here when the same error message is used in multiple locations. But I'm OK with leaving it as it is too.

Same for `ComparisonError` below this one in the code, also only used in a single location.

Reply:
@raph-luc it's a specific error message bound to a specifc error type. The idea is that the code that raises the error is no more verbose than necessary.

Reply:
I think this is resolved now too.